src/rs/src/skill.rs

```diff
@@ -1554,20 +1554,26 @@ impl Skill {
 				if ctx.get_kind(t) != Kind::Player {
 					Skill::destroy.proc(ctx, c, t, data);
 				}
-				let is_open = ctx.cardset() == CardSet::Open;
-				ctx.buffhp(c, 15);
-				if ctx.truehp(c) > 45 {
+				let mut hp_limit = 0;
```

Comment:
This was intentionally made to have legacy/open now agree, in general code should be trusted before card text

Reply:
I will prioritize code logic over text...but I don't quite agree with the principle, as it is far easier to mangle code than text.

Will update card text to increase 15hp per use, and remove/die after 46+